jetty-util/src/main/java/org/eclipse/jetty/util/ssl/KeyStoreScanner.java

```diff
@@ -104,6 +104,16 @@ public void fileRemoved(String filename)
             reload();
     }
 
+    @ManagedOperation(value = "Scan for changes in the SSL Keystore", impact = "ACTION")
+    public void scan()
+    {
+        if (LOG.isDebugEnabled())
+            LOG.debug("scanning");
+
+        _scanner.scan();
+        _scanner.scan();
```

Comment:
We scan twice here because the `Scanner` won't report changes on the first scan, it wants to make sure the file is "stable" before it reports the change, so it needs two consecutive scans with the file unchanged.